terraform/modules/signon_bearer_token/main.tf

```diff
@@ -19,6 +19,14 @@ resource "aws_secretsmanager_secret" "bearer_token" {
   # deletion). Cannot delete and apply Terraform without this:
   # https://github.com/hashicorp/terraform-provider-aws/issues/5127
   recovery_window_in_days = 0
+  
+  tags = merge(
+    var.additional_tags,
+    {
+      name      = "${var.workspace}-workspace-bearer_token"
```

Comment:
I wonder if we shouldn't add the `signon` name in the tag.

Reply:
i will add signon